File: FitnessWeb.API/ViewModels/TrainingViewModel.cs
namespace FitnessWeb.API.ViewModels
{
    public class TrainingViewModel
    {
        public int Id { get; set; }
        public string Type { get; set; }
        public List<ExerciseViewModel> Exercises { get; set; }
        public int Duration { get; set; }
        public string FitnessProgramName { get; set; }

    }
}

File: FitnessWeb.Frontend/AngularApp/src/viewModels/trainingViewModel.ts
import { ExerciseViewModel } from "./exerciseViewModel";

export class TrainingViewModel {
    id: number;
    type: string;
    exercises: ExerciseViewModel[];
    duration: number;
    fitnessProgramName: string;

    constructor(type: string, exercises: ExerciseViewModel[], duration: number, fitnessProgramName: string, id: number) {
        this.id = id;
        this.type = type;
        this.exercises = exercises;
        this.duration = duration;
        this.fitnessProgramName = fitnessProgramName;
    }

}
